examples/material_gallery/test/smoke_test.dart

```diff
@@ -95,18 +68,19 @@ void main() {
 
       final List<double> scrollDeltas = new List<double>();
       double previousY = tester.getTopRight(find.text(demoCategories[0])).y;
-      for (String name in demoNames) {
-        final double y = tester.getTopRight(find.text(name)).y;
+      List<String> routeNames = material_gallery.kRoutes.keys.toList();
+      for (String routeName in routeNames) {
+        final double y = tester.getTopRight(byRouteName(tester, routeName)).y;
         scrollDeltas.add(previousY - y);
         previousY = y;
       }
 
       // Launch each demo and then scroll that item out of the way.
-      for (int i = 0; i < demoNames.length; i += 1) {
-        final String name = demoNames[i];
-        print("$name");
-        smokeDemo(tester, name);
-        tester.scroll(find.text(name), new Offset(0.0, scrollDeltas[i]));
+      for (int i = 0; i < routeNames.length; i += 1) {
+        final String routeName = routeNames[i];
+        print("$routeName");
```

Comment:
tests should be silent. we should find some other way to report transient progress. maybe test supports something?